packages/ERTP/src/types.js

```diff
@@ -186,13 +182,16 @@
 /**
  * @callback MakeIssuerKit
  * @param {string} allegedName
- * @param {MathHelpersName} mathHelperName
+ * @param {AmountMathKind} amountMathKind
  * @returns {IssuerKit}
  *
- * The allegedName is useful for debugging and double-checking
+ * @property {(allegedName: string, amountMathKind: AmountMathKind) => IssuerKit} makeIssuerKit
```

Comment:
It looks like we are double-defining makeIssuerKit. We should remove the `property` one.

Reply:
done